CHANGES.md

```diff
@@ -6,6 +6,16 @@ list](https://github.com/trinodb/trino-python-client/tags), the
 [README](https://github.com/trinodb/trino-python-client/blob/master/README.md)
 and the [PyPI page](https://pypi.org/project/trino/).
 
+## Release 0.325.0
+
+* Queries with parameters now uses `EXECUTE IMMEDIATE` instead of `PREPARE`,
+  `EXECUTE` and `DEALLOCATE` if the server supports `EXECUTE IMMEDIATE`. If you
+  want to revert to older mechanism set the kwarg `legacy_prepared_statements`
+  to `True`.
+  ([#375](https://github.com/trinodb/trino-python-client/issues/375))
+* Update minimum required `requests` library version to `2.31.0`.
```

Comment:
nit: We didn't mention the library bump in the past release notes. Do we start mentioning such changes from this version?

Reply:
I agree. Let me remove this. I included this initially because if anyone has a constraint in their own `requirements.txt` which doesn't satisfy `request>=2.31.0` then the install would fail. However I think that's something for the user to fix since pip gives proper error message in such case.